OracleSiebel/OEL/build/multi-stage/cache/dockerfile

```diff
@@ -0,0 +1,81 @@
+# Globals
+
+ARG VERSION=19.03
+ARG BUILD_DATE=2019-04-30
+
+
+FROM oraclelinux:latest as temp
```

Comment:
Please switch to `oraclelinux:7-slim` (preferred) or at least `oraclelinux:7` as `oraclelinux:latest` will switch to Oracle Linux 8 fairly soon

Reply:
Done